server/api/users.js

```diff
@@ -1,69 +1,32 @@
 const router = require('express').Router();
 const {User, Order, OrderProduct, Product} = require('../db/models');
-const {isAdmin} = require('./routeProtections');
+const {isLoggedIn, isAdmin} = require('./routeProtections');
 module.exports = router;
 
-//ACCOUNT PAGE FOR USER
-//(a bit more secure)
-router.get('/', async (req, res, next) => {
+// GET specific user or Admin user
+router.get('/', isLoggedIn, async (req, res, next) => {
   try {
-    if (req.user.id) {
-      const specificUser = await User.findByPk(req.user.id, {
-        attributes: {
-          exclude: ['password', 'salt']
-        }
+    if (req.user.isAdmin) {
+      const users = await User.findAll({
+        attributes: ['id', 'email']
       });
-      res.json(specificUser);
+      res.json(users);
     } else {
-      //unathorized code
       res.sendStatus(401);
```

Comment:
Need a condition to get a logged in user's profile. Final else: send the status code..